EventFilter/EcalRawToDigi/plugins/UnpackGPU.cu

```diff
@@ -1,5 +1,5 @@
 #include "EventFilter/EcalRawToDigi/interface/ElectronicsIdGPU.h"
-#include "EventFilter/EcalRawToDigi/interface/UnpackGPU.h"
+#include "EventFilter/EcalRawToDigi/plugins/UnpackGPU.h"
```

Comment:
could you use just the file name ?
```suggestion
#include "UnpackGPU.h"
```

this way it's easier to spot problems of dependencies on plugins .